netlogo-gui/src/main/app/tools/Preference.scala

```diff
@@ -38,4 +39,20 @@ object Preference {
       prefs.put("user.country", chosenLocale.getCountry)
     }
   }
+
+  class LineNumbers(tabs: TabsInterface) extends Preference {
+    val i18nKey = "editorLineNumbers"
+    val component = new JCheckBox()
+    val restartRequired = true
```

Comment:
Restart is not required, you just have to hit "Apply".

This experience made me think: do we want that "Apply" button or should changes immediately apply as they're changed? (The second option is more appealing to me. In general, you want the feedback to be as immediate as possible)